libraries/standard/http/cbmc/proofs/HTTPClient_AddRangeHeader/Makefile

```diff
@@ -12,7 +12,7 @@ REMOVE_FUNCTION_BODY +=
 UNWINDSET += strncmp.0:5 convertInt32ToAscii.0:11 convertInt32ToAscii.1:11
 
 PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE).c
-PROOF_SOURCES += $(PROOFDIR)/../../sources/http_cbmc_state.c
+PROOF_SOURCES += $(SRCDIR)/libraries/standard/http/cbmc/sources/http_cbmc_state.c
```

Comment:
Why?

Reply:
Check the closed PR. https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1047#discussion_r453844367